src/GRA.Controllers/MissionControl/PerformerManagementController.cs

```diff
@@ -1523,6 +1557,35 @@ public async Task<IActionResult> KitDetails(KitDetailsViewModel model)
             return View(model);
         }
 
+        [HttpPost]
+        public async Task<IActionResult> DeleteKitSelection(KitSelectionsViewModel model)
+        {
+            var settings = await _performerSchedulingService.GetSettingsAsync();
+            var schedulingStage = _performerSchedulingService.GetSchedulingStage(settings);
+            if (schedulingStage == PsSchedulingStage.Unavailable)
+            {
+                return RedirectToAction(nameof(PerformerManagementController.KitSelections),
+                    new { id = model.Kit.Id });
+            }
+            try
+            {
+                var branchSelection = await _performerSchedulingService
+                    .GetBranchProgramSelectionByIdAsync(model.BranchSelectionId);
+                await _performerSchedulingService.DeleteBranchSelectionAsync(branchSelection);
+                var branch = await _siteService.GetBranchByIdAsync(branchSelection.BranchId);
+                var kit = await _performerSchedulingService.GetKitByIdAsync(model.Kit.Id);
+                ShowAlertSuccess($"{branch.Name}'s selection of \"{kit.Name}\" is deleted!");
+            }
+            catch (GraException gex)
+            {
+                ShowAlertDanger("Unable to delete selection: ", gex);
+                _logger.LogError("Error deleting branch selection Id" +
+                    $"{model.BranchSelectionId} by user {GetId(ClaimType.UserId)}: {gex}", gex);
```

Comment:
LogError should use arguments to set the variables instead of string interpolation
Example: 

    _logger.LogError(gex,
        "Problem displaying index for user {GetActiveUserId}: {Message}",
        GetActiveUserId(),
        gex.Message);

Reply:
Fixed in 9bf3f38be7dda584a234205340b36076a97d5cc0